docs/reporting/index.md

```diff
@@ -0,0 +1,57 @@
+---
+layout: default
+title: Reporting
+nav_order: 34
+has_children: true
+has_toc: false
+---
+
+# Reporting
+
+The Kibana **Reporting** feature allows you to generate, schedule and share reports on-demand or based on an automated schedule. On-demand reports can be instantly downloaded in PNG, PDF and CSV formats, while schedule-based reports can be configured and set up for automated periodic delivery to stakeholders. You can do this either from the **Reporting** tab in left navigation panel or from within the **Dashboard**, **Visualization** and **Discover** tabs. To download, schedule and share reports, you must have the required role permissions. 
```

Comment:
Active voice.

The Kibana reporting feature allows you to generate, schedule, and share reports. You can download reports on-demand in PNG, PDF, and CSV formats and schedule reports for periodic email delivery. To get started, choose **Reporting**.

"To download, schedule and share reports, you must have the required role permissions."

This seems like an entire section of these docs. What permissions do I need to have? Are there combinations of permissions that are applicable (read-only, admin, write, etc.)?

Reply:
I corrected the first part. For role permissions, they're working on integrating this plug-in with the security plug-in. I'm not sure what the details are yet. Maybe Ashwin and follow-up and have this completed. It is due for release only in January.